config.xml

```diff
@@ -77,4 +77,8 @@
     <icon src="resources/android/icon/drawable-xhdpi-icon.png" />
     <plugin name="cordova-plugin-bluetoothle" spec="https://github.com/sgrebnov/cordova-plugin-bluetoothle.git" />
     <plugin name="cordova-open-zwave" version="0.0.1" src="https://github.com/vladimir-kotikov/cordova-open-zwave" />
+    <feature name="InAppBrowser">
+        <param name="ios-package" value="CDVInAppBrowser" />
+        <param name="android-package" value="org.apache.cordova.inappbrowser.InAppBrowser" />
```

Comment:
This is per the instructions here: https://auth0.com/docs/quickstart/native/ionic2 and configures the cordova "inappbrowser" plugin which shows the Auth0 UX. I read the docs for this plugin and it does not seem to require any additional configuration for Windows (I also built and tested the app with these changes on Windows 10 and the plugin worked as expected)